libraries/Microsoft.Bot.Builder.Ai/LocaleConverterMiddleware.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using Microsoft.Bot.Schema;
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Ai
+{
+    /// <summary>
+    /// Middleware to convert messages between different locales specified
+    /// </summary>
+    public class LocaleConverterMiddleware : IMiddleware
+    {
+        private ILocaleConverter localeConverter; 
```

Comment:
readonly

Reply:
2 of your member variables are "_" prefixed. 2 are not. 

The general pattern we've used in to prefix using "_" for private member variables.

Reply:
Updated